frame/contracts/src/wasm/runtime.rs

```diff
@@ -921,7 +994,7 @@ define_env!(Env, <E: Ext>,
 	// `ReturnCode::KeyNotFound`
 	[seal0] seal_get_storage(ctx, key_ptr: u32, out_ptr: u32, out_len_ptr: u32) -> ReturnCode => {
```

Comment:
Should probably be factored out to its own function on `Runtime` just as for `seal_set_storage`.